libs/coin-modules/coin-bitcoin/src/getTransactionStatus.spec.ts

```diff
@@ -0,0 +1,125 @@
+/* eslint @typescript-eslint/consistent-type-assertions: 0 */
+
+import { Account } from "@ledgerhq/types-live";
+import * as cache from "./cache";
+import { BitcoinInput, Transaction } from "./types";
+import getTransactionStatus, { MAX_BLOCK_HEIGHT_FOR_TAPROOT } from "./getTransactionStatus";
+import {
+  AddressesSanctionedError,
+  AddressesSanctionedErrorAttributes,
+} from "@ledgerhq/coin-framework/sanction/errors";
+import BigNumber from "bignumber.js";
+import * as sanction from "@ledgerhq/coin-framework/sanction/index";
+
+const calculateFeesSpy = jest.spyOn(cache, "calculateFees");
+const validateRecipientSpy = jest.spyOn(cache, "validateRecipient");
+const isAddressSanctionedSpy = jest.spyOn(sanction, "isAddressSanctioned");
+
+describe("getTransactionStatus on Bitcoin", () => {
+  it("should return as sender error only sanctioned utxo addresses", async () => {
+    validateRecipientSpy.mockResolvedValue({
+      recipientError: undefined,
+      recipientWarning: undefined,
+    });
+
+    const sanctionedAddresses = [
+      "bc1qneyw3q5r7m88e7yheem7wn4wvx2chug5g4hrve",
+      "bc1qx47dkcps02ce4e9ppkse4kesx79ds3h4zsadqa",
+    ];
+
+    const utxoAddresses = [
+      {
+        address: "bc1qneyw3q5r7m88e7yheem7wn4wvx2chug5g4hrve",
+      },
+      {
+        address: "bc1qx47dkcps02ce4e9ppkse4kesx79ds3h4zsadqa",
+      },
+      {
+        address: "bc1q2nj06sukhuuuvc88g0rh7eqe3r9n3rm66a7syv",
+      },
+      {
+        address: "bc1qa8kyt89kfh2cnf0q39pglhegj53xcwqtkf6jkd",
+      },
+    ] as unknown as BitcoinInput[];
+
+    isAddressSanctionedSpy.mockImplementation((_, address: string) => {
+      return Promise.resolve(sanctionedAddresses.some(sanctioned => address === sanctioned));
```

Comment:
```suggestion
      return Promise.resolve(sanctionedAddresses.inculdes(address));
```
More popular I think

Reply:
done